src/expect.wren

```diff
@@ -106,15 +106,22 @@ class Expect {
 
     // Error tests
     #!deprecated
-    abortsWith(err) { toAbortWith(err) }
+    abortsWith(errorMessage) { toAbortWith(errorMessage) }
 
-    toAbortWith(err) {
+    toAbortWith(expectedMessage) {
         var f = Fiber.new { _value.call() }
         var result = f.try()
-        assert(result.toString == err, "Expected error '%(err)' but got %(result)")
+        var errorMessage = f.error
```

Comment:
We really need to check `Fiber.error` not the result of `try()`, which can be a valid return value from the fiber (not an error at all)... not sure if this was my original mistake or not, quite possibly was.